index.js

```diff
@@ -168,3 +168,64 @@ exports.toGregorian = function (hy, hm, hd) {
     var g = d2g(h2d(hy, hm, hd));
     return g;
 }
+
+function getDateSplit(fullDate) {
+    const arr = fullDate.split('-');
+    const year = (arr)[0];
+    const month = (arr)[1];
+    const day = (arr)[2];
+    return {
+      year,
+      month,
+      day,
+    };
+}
+
+function isDateMatch(hd) {
+    const dateReg = /^\d{4}([./-])\d{2}\1\d{2}$/;
+    const isMatch = `${hd}`.match(dateReg) !== null;
+    if (isMatch) {
+      return true;
+    }
```

Comment:
We can do it like this also 
isMatch ? true : false 
:stuck_out_tongue_closed_eyes: